doc/listeners/listen-c2s.md

```diff
@@ -160,6 +178,6 @@ The following section configures two C2S listeners.
 ```
 
 * One at port 5222, which accepts a plain TCP connection and allows to use StartTLS for upgrading it to an encrypted one. The files containing the certificate and the DH parameter are also provided.
-* One at port 5223, which accepts only encrypted TLS connections - this is the legacy method as StartTLS is preferred.
+* One at port 5223, which accepts only encrypted TLS connections - this is the legacy method as StartTLS is preferred. Also known as Direct TLS.
```

Comment:
Update this whole line if you're already touching it, it is the other way around, starttls is the worst option (by faaaaaaar).

Reply:
sssssh, common, 5222 allows downgrade attacks, if hacker controls the channels and the client is naive enough, to believe that server does not support TLS. _see how useful it could be :D_